src/static/js/modules/nemo.js

```diff
@@ -130,7 +132,7 @@ $(document).ready(function (){
                     else {
                         thanks = "Thanks, weâ€™ll be in touch.";
```

Comment:
Good catch. I had figured there was a reason for it, and decided not to touch it.